lib/ddtrace/contrib/rack/patcher.rb

```diff
@@ -39,51 +39,12 @@ def patch
             @patched = true
           end
 
-          if (!instance_variable_defined?(:@middleware_patched) || !@middleware_patched) \
```

Comment:
I take it this feature effectively replaces the middleware names option we had?

Reply:
Yup.